### PR TITLE
Documentation for updating docker images

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

File: src/nhd_stream_tile_creation.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+#to create NHDPlus stream order tiles
+tippecanoe -zg --detect-shared-borders --simplify-only-low-zooms --drop-densest-as-need --simplification=5 --output-to-directory nhd_order_grouped nhdplus_order_1.geojson nhdplus_order_2_3.geojson nhdplus_order_4_5.geojson nhdplus_order_6.geojson
```
Comment (wdwatkins) [inline src/nhd_stream_tile_creation.sh]: This is in a Jenkins job now, right?

Comment (mhines-usgs) [reply src/nhd_stream_tile_creation.sh]: yes, it got modified slightly to new geojsons but it's captured here more or less https://github.com/usgs-makerspace/wbeep-processing/blob/master/jenkins/create_rivers_basemap#L45

File: README.md [hunk 2]
```diff
@@ -73,3 +73,20 @@ We are doing this to be able to troubleshoot the viz while the model is still ru
...
+
+## Updating the Docker images
+
+There are two docker images used in the processing steps --- one for R, and one for tippecanoe.  The built images are stored in the Docker registry at https://code.chs.usgs.gov/wma/iidd/wbeep-data-processing.  If you need to update the images, follow these steps:
+
+1. Make the appropriate changes to the Dockerfile, and make sure it builds locally by running `docker-compose build` in the terminal from the directory containing both the Dockerfile and docker-compose.yml.  
+
+2. Make a pull request and have it reviewed.  
+
+3. When the pull request is merged, it should trigger a Jenkins job (watching only the Dockerfile in the repo) that will build the image and push it to the docker registry tagged as `R-latest` or `tippecanoe-latest`.
```
Comment (mhines-usgs) [inline README.md]: doesn't seem like this job is triggered based on merges into the wbeep-processing repo right now, but I could change that if we want that to be monitoring the repo?

Comment (mhines-usgs) [reply README.md]: actually neither, i'll set them up to poll

Comment (wdwatkins) [reply README.md]: Cool.  We'll probably just want to revisit this if we decide to start uniquely tagging images

**Pull request merged**